system-tests/project-fixtures/react/cypress-vite.config.ts

```diff
@@ -9,12 +10,25 @@ declare global {
   }
 }
 
+const port = 8888
+
+const viteConfig = require('./webpack.config.js')
```

Comment:
@mschile did you mean to require the webpack config here?

Reply:
No, that's what I get for changing it right at the end. Updated to vite. [0bda09b](https://github.com/cypress-io/cypress/pull/27677/commits/0bda09bf8d9a2fea81a013a150e57defe44f7c74)